yaksh/models.py

```diff
@@ -116,6 +118,9 @@ def activate(self):
     def deactivate(self):
         self.active = False
 
+    def add_teachers(self, *users):
+        self.teachers.add(*users)
```

Comment:
Please just make sure that the users that are being added have moderator permissions
(Maybe an IF condition that checks this and returns appropriate value)